runtime/src/main/java/org/corfudb/protocols/wireprotocol/TailsResponse.java

```diff
@@ -18,19 +21,24 @@
 
 @Data
 @RequiredArgsConstructor
+@AllArgsConstructor
 public class TailsResponse implements ICorfuPayload<TailsResponse> {
 
+    long epoch = Layout.INVALID_EPOCH;
+
```

Comment:
This is not needed anymore, right?

Reply:
Still needed. To return from getTails from Utils.

Reply:
This is serialized and being sent to the server, but the server doesn't use this value.